change/@fluentui-react-charting-bda51650-008f-417c-ab0d-62e7dd4a22fd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Lets update the PR with description and performance numbers.

Reply:
Done